src/Shared/Recruit.Vacancies.Client/Domain/Entities/ApplicationReview.cs

```diff
@@ -15,5 +15,11 @@ public class ApplicationReview
         public DateTime? StatusUpdatedDate { get; set; }
         public VacancyUser StatusUpdatedBy { get;set; }
         public string CandidateFeedback { get; set; }
+        public bool IsWithdrawn { get; set; }

```

Comment:
Withdrawn is just another state, why not use the Status field?

Reply:
I started down this path originally. An application can be both Successful and Withdrawn for example.  I think its useful to know what the state was before the withdrawal.

Reply:
one is ReviewStatus the other is ApplicationStatus. IsWithdrawn sounds fine to me